### ISSUE TITLE
salad command isn't working

### ISSUE THREAD

**Issue by santagada** [salad/salad#62]
If I use the salad command to run tests it doesn't start firefox:

```
$ salad
salad - WARNING - Error starting up firefox
```

and then errors out. If I try to pass --browser:

```
$ salad --browser chrome
Usage: salad or type salad -h (--help) for help

salad: error: no such option: --browser
```

If I then create a steps and terrains and import everything tests work with lettuce, but then I can't choose browser right?

**Comment by beanqueen** [comment]
hi, could you try out my fork or my branch in this repo? also, i could help you with the debugging, but then write me an email to jana@rekittke.name

**Comment by santagada** [comment]
It is working, can we release a new version of the package?

**Comment by skoczen** [comment]
Hey @santagada (and @beanqueen), thanks for reaching out with this.  Here's all the info, with a TL;DR and a summary of what you can do going forward.

## Making a new release

There's a huge discussion on releasing it over on #60.

TL;DR: the fork contains significant (and some backwards incompatible) changes, there are outstanding questions on the code, and the philosophy is different enough (shifting from a BDD framework to more of a QA-automation framework), that I'm honestly torn on what to do.

Compounding that is a situation where one either I or one the maintainers (a long time ago) merged in a PR that wasn't ready - so master, which has the `--browser` option and some fixes has the wrong version numbers, contact info, etc.

Two options that are on the table:

1. That PR gets to a place where all parties are happy with it being merged and released, or 
2. That PR becomes a separate project, that takes on its own identity, code standard, etc.

I've been thinking about this for a couple months and haven't come to any real conclusion.  My open-source time is also mostly swallowed by maintaining will, polytester, django-seo-js, and django-ajax-uploader, and these days, I mostly use protractor for functional testing while at work because of its angular smarts.

Thus the conundrum.  As an end-user, your thoughts are welcome!

## Getting yourself rolling.

Your two choices at this point are:
1. Installing from master here, and dealing with the wonky version number, or
2. Installing and using @beanqueen 's fork.

The fork is better maintained, but less well documented.

I wish I had better answers for you, but as I'm sure you already know, open-source is messy sometimes.  This, unfortunately, is one of them.

Thanks for your patience.

-Steven
